edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -96,10 +97,12 @@ impl DockerModuleRuntime {
     }
 }
 
-fn get_base_path(url: &Url) -> &str {
+fn get_base_path(url: &Url) -> Result<PathBuf> {
     match url.scheme() {
-        "unix" => url.path(),
-        _ => url.as_str(),
+        "unix" => url
```

Comment:
`"unix" => Ok(url.to_uds_file_path()?),`

Reply:
Ah, ok. I didn't realize the `?` implicitly coerces the error type. I'll update this.